client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/SubmissionEditStepForm.jsx

```diff
@@ -287,8 +287,8 @@ class SubmissionEditStepForm extends Component {
   }
 
   renderFinaliseButton() {
-    const { intl, attempting, allCorrect, isSaving } = this.props;
-    if (attempting && allCorrect) {
+    const { intl, attempting, allCorrect, isSaving, allowPartialSubmission } = this.props;
+    if (attempting && (allowPartialSubmission || allCorrect)) {
```

Comment:
We changed `all_correct` on the backend to `considered_correct`. Should change the name of this variable also for consistency.

Reply:
This frontend `allCorrect` means something else compared to backend. A more apt name may be `allAnswersConsideredCorrect`.
Defined at
https://github.com/Coursemology/coursemology2/blob/master/client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/index.jsx#L97